Source/Scene/Model.js

```diff
@@ -848,22 +845,11 @@ define([
         byteOffset += sizeOfUint32; // Skip length
 
         var sceneLength = view.getUint32(byteOffset, true);
-        byteOffset += sizeOfUint32;
-
-        var sceneFormat = view.getUint32(byteOffset, true);
-        byteOffset += sizeOfUint32;
+        byteOffset += sizeOfUint32 + sizeOfUint32; // Skip length
```

Comment:
Change the comment to `// Skip sceneFormat`.

Refer to the header's format: https://github.com/KhronosGroup/glTF/blob/master/extensions/Khronos/KHR_binary_glTF/README.md#binary-gltf-layout

Also, can you give me commit access to your fork so I can make trivial tweaks like this for you?